grammar.js

```diff
@@ -1105,6 +1106,13 @@ module.exports = grammar({
       ')'
     ),
 
+    class_static_block: $ => seq(
+      'static',
+      '{',
+      repeat($.statement),
+      '}',
```

Comment:
Can you reuse the `statement_block` rule here?

Reply:
Done